app/src/main/java/com/keylesspalace/tusky/util/AttachmentHelper.kt

```diff
@@ -27,10 +28,16 @@ private fun formatDuration(durationInSeconds: Double): String {
 }
 
 fun List<Attachment>.aspectRatios(): List<Double> {
+    val displayMetrics = Resources.getSystem().displayMetrics
+    val deviceIsHigher = displayMetrics.heightPixels > displayMetrics.widthPixels
+
+    val minAspect = if (deviceIsHigher) 0.5 else 1.2
+    val maxAspect = if (deviceIsHigher) 2.0 else 3.0
```

Comment:
Why specifically these constants for landscape?

Reply:
I tried to reverse "the idea of 0.5 to 2 for a portrait phone". However the literal 1 to 4 is probably a bit too much.

So these are "best and simple guesses for a wider range of landscape".

Generally one could discuss if this should be relative to the actual aspect ratio value of the device. (After all simply "height > width ?" is somewhat generic)

However I would venture that this is a simple solution.

(1.2 and not 1 because at least phones are nowadays much much higher than wide.)